styles/RolePlayGateway/template/characters_profile_body.html

```diff
@@ -52,104 +60,111 @@ <h3 style="margin-top:5px;">
         <!-- IF CHARACTER_HISTORY -->
         <h3>History</h3>
         <p style="text-align:justify;">{CHARACTER_HISTORY}</p>
-        <!-- IF CHARACTER_STATS -->
+        <!-- ENDIF -->
+        <!-- IF S_IS_BETA -->
         <h3>Statistics</h3>
-        <b><u>Body</u></b><br/>
+        <b><u>Body</u></b><br>
         <b>Power</b>
-          <select>
-            <option value="F RANK">F (-8)</option>
-            <option value="D RANK">D (-4)</option>
-            <option selected ="selected" value="C RANK">C (+0)</option>
-            <option value="B RANK">B (+4)</option>
-            <option value="A RANK">A (+8)</option>
-            <option value="S RANK">S (+12)</option>
+          <select name="power">
+            <option title="Power F RANK" value="-8">F (-8)</option>
```

Comment:
This is a great insight — instead of maintaining a map of "names" -> "values", we can just specify the values at this level it becomes configurable later.  Nice work!